build.gradle

```diff
@@ -249,6 +277,54 @@ task deploy() {}
 
 subprojects {
 
+  if (file('src/main/java').directory) {
+    apply plugin: 'com.jfrog.bintray'
+    apply plugin: 'maven-publish'
+
+    publishing {
+      publications {
+        mavenJava(MavenPublication) {
+          groupId "tech.pegasys.${project.group}"
+          version "${project.version}"
+          from components.java
+          artifact sourcesJar
+          artifact javadocJar
+          versionMapping {
+            usage('java-api') { fromResolutionOf('runtimeClasspath') }
+            usage('java-runtime') { fromResolutionResult() }
+          }
+          pom {
+            name = "Pantheon - ${project.name}"
+            url = 'http://github.com/PegaSysEng/pantheon'
+            licenses {
+              license {
+                name = 'The Apache License, Version 2.0'
+                url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+              }
+            }
+            scm {
+              connection = 'scm:git:git://github.com/PegaSysEng/pantheon.git'
+              developerConnection = 'scm:git:ssh://github.com/PegaSysEng/pantheon.git'
+              url = 'https://github.com/PegaSysEng/pantheon'
+            }
+          }
+        }
+      }
```

Comment:
I think if you add a `repository` block you may get the maven metadata

```gradle
  repositories {
    maven {
      // change URLs to point to your repos, e.g. http://my.org/repo
      def releasesRepoUrl = "$buildDir/repos/releases"
      def snapshotsRepoUrl = "$buildDir/repos/snapshots"
      url = version.endsWith('SNAPSHOT') ? snapshotsRepoUrl : releasesRepoUrl
    }
  }
```

Reply:
No luck.  But I've tested with both gradle and maven depending on these publish jars and it works even without the metadata.